net/WebSocketHandler.hpp

```diff
@@ -784,7 +784,10 @@ class WebSocketHandler : public ProtocolHandlerInterface
                   [[maybe_unused]] unsigned char flags, bool flush = true) const
     {
         if (!socket || data == nullptr || len == 0)
+        {
+            LOG_DBG("Socket or data missing. Cannot send WS frame");
```

Comment:
I wonder how noisy this might be in some cases. We have to keep an eye, in case it spams when we shutdown a socket or some edge-case like that.

Regarding insertfile, that command shouldn't have anything special that relates to the socket's functionality. I seriously doubt if this log will help at all. It's unlikely that everything is perfectly fine and we lose the socket only when insertfile is executed.